src/lib/ext-monitor.js

```diff
@@ -1,6 +1,9 @@
 import { getActivityLogPageURL } from './ext-listen.js';
+import { load } from './save-load.js';
 
 export default class ExtensionMonitor {
+  // Map<number, Array>
+  loadedLogs = new Map();
```

Comment:
The inline comments does explicitly mention what is the type of the key, but not what it does represents, and the name of the property does not make it clear neither.

Calling this property `loadedLogsByTabId` would be making it clear enough without having to spelling that again in the inline comment.
The property is used only in 4 places and so even if the name is a bit longer it shouldn't make the core too much more verbose.

Otherwise you should at least mention it nearby the inline comment.